src/mame/misc/babysuprem.cpp

```diff
@@ -14,11 +14,24 @@
      1 x 8-dip + 1 x 4 dip banks
 
     There's a complete manual with schematics at https://www.recreativas.org/manuales
+    Note there are a nuumber of errors in the manual/schematic.
+	
+	There is a basic layout file which makes the game playable.
+
+    TODO: Requires a proper layout file with artwork.
+          Add watchdog and determine timeout.
+		  There are a six lamps whose use needs identifying.
+		  Identify what's being accessed in the i/o map, this doesn't appear to
+		  affect the game at all.
+		  The manual lists all chips used and includes a Z80 DART and Z80 CTC neither
+		  of which appear in the schematic.
 */
 
 #include "emu.h"
-
+#include "babysuprem.lh"
```

Comment:
Please put “headers” that actually contain definitions after all the declaration headers.  Also, the blank lines separating grouped headers are intentional.

Reply:
OK, didn't realise about the blank lines, some drivers have them and others don't. Will fix.